serverless.yml

```diff
@@ -37,6 +37,7 @@ custom:
   u2fTable: ${self:custom.namespace}_u2f
   dev_env: staging
   prod_env: production
+  secondaryRegion: ${env:AWS_REGION_SECONDARY, "us-west-2"}
```

Comment:
Just verifying if this is correct to default to.

Reply:
Yep. I believe our current general plan is to put our primary AWS infrastructure in us-east-2 (to reduce the chances of both our infrastructure and the us-east-1 control planes being down at the same time) and to put our secondary infrastructure in us-west-2 (because it's cheaper than us-west-1). Thanks for checking.